src/main/java/uk/nhs/tis/trainee/usermanagement/api/UserAccountResource.java

```diff
@@ -110,4 +111,22 @@ ResponseEntity<Void> resetUserAccountMfa(@PathVariable String username) {
     log.info("MFA reset for user '{}'.", username);
     return ResponseEntity.noContent().build();
   }
+
+  /**
+   * Delete TSS Cognito account for the given user.
+   *
+   * @param username The username of the user.
+   * @return 204 No Content, if successful.
+   */
+  @DeleteMapping("/delete/{username}")
```

Comment:
Actually, now it's changed to a `DELETE` request method we probably don't need it in the path anymore, so just ` DELETE /api/user-account/{username}`... sorry 😅

Reply:
It's a good shot. Thank you.